portal-kernel/src/com/liferay/exportimport/kernel/lar/BasePortletDataHandler.java

```diff
@@ -477,50 +477,6 @@ public void prepareManifestSummary(
 		}
 	}
 
-	/**
```

Comment:
@dantewang @yuhai 
This deprecation is weird according to the commit history, only implementation was deprecated, but not the interface method, also, this deprecation was done in a ticket which is dealing with exception handling, and all other logic is related to exception handling except this deprecation. Thus I can not tell if this was done on purpose or by accident. 

Let's hold on for this one, and move on with others.

Tina